proarc-common/src/main/java/cz/cas/lib/proarc/common/catalog/AlephXServer.java

```diff
@@ -17,8 +17,19 @@
 package cz.cas.lib.proarc.common.catalog;
 
 import cz.cas.lib.proarc.common.config.CatalogConfiguration;
+import cz.cas.lib.proarc.common.config.CatalogQueryField;
 import cz.cas.lib.proarc.common.mods.ModsUtils;
 import cz.cas.lib.proarc.common.xml.Transformers;
+import org.w3c.dom.Element;
+
+import javax.xml.bind.JAXB;
```

Comment:
Na abecední řazení importů hledám řešení pro Idea IDE (nativně podporuje pouze určení pořadí konkrétních balíků - tj musely by se vyjmenovat veškeré použité balíky), do té doby řazení v IDE vypnu.